pkgs/development/compilers/llvm/11/libc++/default.nix

```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation {
   buildInputs = [ libcxxabi ] ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
 
   cmakeFlags = [
-    "-DLIBCXX_LIBCPPABI_VERSION=2"
+    "-DLIBCXX_STANDALONE_BUILD=OFF"
```

Comment:
This doesn't work on Darwin (and elsewhere), because the build directory has the wrong name.
It is `libcxx-11.0.0.src`, but should be `libcxx` according to
```
CMakeLists.txt(31):  if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR OR LIBCXX_STANDALONE_BUILD )
```

Reply:
This is redundant, removed in 487cd18f322cdbcf6ffab0c9e9ea0da1456f45f9.